.github/workflows/integration-rate-limiter.yaml

```diff
@@ -1,18 +1,28 @@
 name: Cloud Hypervisor Tests (Rate-Limiter)
 on:
+  merge_group:
+  pull_request:
   push:
     branches:
       - main
```

Comment:
We don't need to run rate-limiter (and VFIO/SGX) on main branch any more, given we have it covered from merge queue. Right?